src/services/tasks/api/index.ts

```diff
@@ -0,0 +1,76 @@
+import { ApiService } from './types';
+import { HttpInstance } from '../../../http/types';
+import { HttpClient } from '../../http-client';
+import { OptionsBase } from '../../types';
+
+export default (client: HttpClient, httpAuth: HttpInstance): ApiService => ({
+  async get<T>(name: string, path: string, options: OptionsBase): Promise<T> {
+    const { data } = await client.get(
+      httpAuth,
+      `/api/${name}/${path}`,
```

Comment:
So for all of these methods the path can either start with a / e.g `/users/:id` or without `users/:id`, it will affect line 10. For now I did it without but I'm super happy that it could change if there's a preference.

Reply:
Discussed, both!